test/js/upload-wizard/file-info-responses.js

```diff
@@ -0,0 +1,293 @@
+export const METADATA_AND_EXPRESSION_FILE_STUDY = {
+  study: {
+    '_id': {
+      '$oid': '60a2b9f4cc7ba082358b5448'
+    },
+    'name': 'Chicken raw counts',
+    'description': 'Synthetic chicken raw counts',
+    'bucket_id': 'fc-458fcddb-bbef-4eb3-b0c6-3d2253df623e',
+    'accession': 'SCP34'
+  },
+  files: [{
+    '_id': {
+      '$oid': '60a2b9fbcc7ba082358b544a'
+    },
+    'created_at': '2021-05-17T14:46:19.400-04:00',
+    'data_dir': '51e9c0d33e9b698b118bbb884fab092a1b2d22a9f4133966010560c60835361a',
+    'description': null,
+    'file_type': 'Metadata',
+    'generation': '1621277203802483',
+    'genome_annotation_id': null,
+    'genome_assembly_id': null,
+    'human_data': false,
+    'human_fastq_url': null,
+    'is_spatial': false,
+    'name': 'metadata.txt',
+    'options': {},
+    'parse_status': 'parsed',
+    'queued_for_deletion': false,
+    'remote_location': '',
+    'spatial_cluster_associations': [],
+    'status': 'uploaded',
+    'study_file_bundle_id': null,
+    'study_id': {
+      '$oid': '60a2b9f4cc7ba082358b5448'
+    },
+    'taxon_id': null,
+    'updated_at': '2021-05-17T14:48:52.287-04:00',
+    'upload_content_type': 'text/plain',
+    'upload_file_name': 'metadata.txt',
+    'upload_file_size': 7065,
+    'use_metadata_convention': true,
+    'version': null,
+    'x_axis_label': '',
+    'x_axis_max': null,
+    'x_axis_min': null,
+    'y_axis_label': '',
+    'y_axis_max': null,
+    'y_axis_min': null,
+    'z_axis_label': '',
+    'z_axis_max': null,
+    'z_axis_min': null,
+    'upload': {
+      'url': '/Users/dbush/code/scp/single_cell_portal_core/data/51e9c0d33e9b698b118bbb884fab092a1b2d22a9f4133966010560c60835361a/60a2b9fbcc7ba082358b544a/original/metadata.txt'
```

Comment:
Nit - It doesn't seem to impact the tests, so can we change these urls?  Seeing a local path from your machine makes my code smell radar go off.